### PR TITLE
Use panToBounds with padding to pan the infowindow for JS Maps API v3.32 and above

### PR DESCRIPTION
**Pull request by florencia-irena**
Using projection.fromLatLngToContainerPixel to compute infowindow panning will not work correctly if the map is currently moving (e.g. setCenter() is called right before the infowindow pan calculation happens) on JS Maps API v3.32 and above.

This can be fixed by using panToBounds() method with padding, which is added in the most recent version (v3.32.8) of JS Maps API.

### CODE REVIEW

**Comment by garylittleRLP** [comment]
Looks good. Can you change line 3 to:

 * @version 1.1.19 [April 6, 2018]

File: infobox/src/infobox.js [hunk 1]
```diff
@@ -333,44 +333,62 @@ InfoBox.prototype.panBox_ = function (disablePan) {
...
+          if (pixPosition.y < (-iwOffsetY + padY)) {
+            yOffset = pixPosition.y + iwOffsetY - padY;
+          } else if ((pixPosition.y + iwHeight + iwOffsetY + padY) > mapHeight) {
+            yOffset = pixPosition.y + iwHeight + iwOffsetY + padY - mapHeight;
+          }
         }
-      }
 
-      if (!(xOffset === 0 && yOffset === 0)) {
+        if (!(xOffset === 0 && yOffset === 0)) {
```
Comment (stephenmcd) [inline infobox/src/infobox.js]: this line shouldn't change (de-indent)

Comment (stephenmcd) [reply infobox/src/infobox.js]: My bad, I see this is correct actually. Please ignore.

**Comment by stephenmcd** [comment]
Thanks all

**Pull request merged**